app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -25,9 +25,13 @@
 
 %>
 <% if card_helper.href || card_helper.extra_details.any? %>
-  <div class="<%= classes %> <%= brand_helper.brand_class %>"
-    <%= "data-module=gem-track-click" if card_helper.is_tracking? %>
-    <%= "lang=#{card_helper.lang}" if card_helper.lang %>>
+  <%
+    classes = "#{classes} #{brand_helper.brand_class}"
+    data_attributes = {}
```

Comment:
This is good. I wonder if it's worth making it more flexible while we're at it (but feel free to disagree) so we could pass anything as data attributes, something like this?

```
data_attributes ||= {}
((data_attributes[:module] ||= "") << " " << "gem-track-click ga4-link-tracker").strip! if card_helper.is_tracking?
```

I've lifted this from the accordion component. Looking at it again I think it might be possible to simplify it to `((data_attributes[:module] ||= "") << " gem-track-click ga4-link-tracker").strip! if card_helper.is_tracking?`.

Obviously this would require adding more tests, and a doc entry, so again, feel free to disagree with me.

Reply:
Has [3156](https://github.com/alphagov/govuk_publishing_components/pull/3156) handled this? If not, could I look at implementing this in a separate branch? Only because I've been holding this PR up for a while and it would be good to get it merged

Reply:
Don't worry about this suggestion, I'm putting together a spike on something that I think will be able to to deal with this.